www/assets/js/components/SearchPage.tsx

```diff
@@ -134,15 +134,21 @@ export default function SearchPage() {
 
   // eslint-disable-next-line react-hooks/exhaustive-deps
   const toggleResourceSearch = useCallback(
-    toggleOn => async () => {
+    nextResourceFilterState => async () => {
+      if (isResourceFilterActive() === nextResourceFilterState) {
+        //Immediately return in case user clicks
+        // and already active facet.
+        //Githun issue https://github.com/mitodl/ocw-hugo-themes/issues/105
```

Comment:
```suggestion
        // Immediately return in case the user clicks an already active facet.
        // Github issue https://github.com/mitodl/ocw-hugo-themes/issues/105
```